lib/simulator-xcode-11.4.js

```diff
@@ -104,6 +105,25 @@ class SimulatorXcode11_4 extends SimulatorXcode11 {
   async clearKeychains () {
     await this.simctl.resetKeychain();
   }
+
+  /**
+   * Opens simulator's UI Client if not already open and boots sim
+   *
+   * @param uiClientPid - process id of simulator UI client.
```

Comment:
You could simply mention `@inherit` if the documentation repeats the overridden one

Reply:
sounds good, using `@inheritdoc tag` that also implies override. https://jsdoc.app/tags-inheritdoc.html